sdk/personalizer/azure-ai-personalizer/azure/ai/personalizer/aio/_patch.py

```diff
@@ -0,0 +1,1161 @@
+        :type feature_importance_id: str
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        await self._client.delete_feature_importance(feature_importance_id, **kwargs)
+
+    def send_request(self, request: HttpRequest, **kwargs: Any) -> Awaitable[AsyncHttpResponse]:
+        """Runs the network request through the client's chained policies.
+
+        >>> from azure.core.rest import HttpRequest
+        >>> request = HttpRequest("GET", "https://www.example.org/")
+        <HttpRequest [GET], url: 'https://www.example.org/'>
+        >>> response = await client.send_request(request)
+        <AsyncHttpResponse: 200 OK>
+
+        For more information on this code flow, see https://aka.ms/azsdk/dpcodegen/python/send_request
+
+        :param request: The network request you want to make. Required.
+        :type request: ~azure.core.rest.HttpRequest
+        :keyword bool stream: Whether the response payload will be streamed. Defaults to False.
+        :return: The response of your network call. Does not do error handling on your response.
+        :rtype: ~azure.core.rest.AsyncHttpResponse
+        """
+        return self._client.send_request(request, **kwargs)
+
+    async def close(self) -> None:
+        await self._client.close()
+
+    async def __aenter__(self) -> "PersonalizerAdministrationClient":
+        await self._client.__aenter__()
+        return self
+
+    async def __aexit__(self, *exc_details) -> None:
+        await self._client.__aexit__(*exc_details)
+
+
+class PersonalizerClient:
+    """Personalizer Service is an Azure Cognitive Service that makes it easy to target content and
+    experiences without complex pre-analysis or cleanup of past data. Given a context and
+    featurized content, the Personalizer Service returns which content item to show to users in
+    rewardActionId. As rewards are sent in response to the use of rewardActionId, the reinforcement
+    learning algorithm will improve the model and improve performance of future rank calls.
+
+    :param endpoint: Supported Cognitive Services endpoint. Required.
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential or ~azure.core.credentials_async.AsyncTokenCredential
+    :keyword api_version: Api Version. Default value is "2022-09-01-preview". Note that overriding
+     this default value may result in unsupported behavior.
+    :paramtype api_version: str
+    """
+
+    def __init__(
+        self, endpoint: str, credential: Union[AzureKeyCredential, AsyncTokenCredential], **kwargs: Any
+    ) -> None:
+        self._client = PersonalizerClientGenerated(
+            endpoint=endpoint.rstrip("/"),
+            credential=credential,  # type: ignore
+            authentication_policy=kwargs.pop("authentication_policy", _authentication_policy(credential)),
```

Comment:
pass through **kwargs into the function

```suggestion
            authentication_policy=kwargs.pop("authentication_policy", _authentication_policy(credential, **kwargs)),
```

Reply:
Fixed.